cfgov/unprocessed/apps/owning-a-home/js/explore-rates/RateCheckerChart.js

```diff
@@ -152,11 +152,11 @@ function RateCheckerChart() {
   function setStatus( state ) {
     if ( state === RateCheckerChart.STATUS_OKAY ) {
       _chartDom.classList.remove( 'warning' );
-      _resultAlertDom.classList.add( 'u-hidden' );
+      _resultAlertDom.classList.remove( 'm-notification__visible' );
       _failAlertDom.classList.add( 'u-hidden' );
     } else if ( state === RateCheckerChart.STATUS_WARNING ) {
       _chartDom.classList.add( 'warning' );
-      _resultAlertDom.classList.remove( 'u-hidden' );
+      _resultAlertDom.classList.add( 'm-notification__visible' );
```

Comment:
This is fine for now, but we'll want to make these actual notifications with JS so the Notification show()/hide() methods are called instead of modifying classes.

Reply:
yeah, I'll switch to that.

Reply:
I left this as-is for now.